### ISSUE TITLE
Create a CI to avoid wrong alias/metadata in Markdowns

### ISSUE THREAD

**Issue by rikatz** [kubernetes/website#30694]
**This is a Bug Report**

Contextualization:

We've seen some past PRs in localized documents that created wrong redirection in Kubernetes Website. The more recent incident was with kubernetes.io/security, where this was redirected to Korean security and was fixed in https://github.com/kubernetes/website/pull/30684

Jim added some more context that this happened on past (see Slack thread https://kubernetes.slack.com/archives/C019LFTGNQ3/p1638221631143800).

The main statement is: we need to create a CI that verifies when a used adds a Markdown file, with aliases containing a prepended slack (like /security) and if this is not part of the English localized docs, this CI should fail.

So, invalid Markdowns are like:

- content/pt-br/..../something.md
```
---
title: Kubernetes XXXX
weight: 10
aliases: [/cve/,/cves/]
---
```

When it should be (without the leading slash):
```
---
title: Kubernetes XXXX
weight: 10
aliases: [cve/,cves/]
---
```

**Proposed Solution:**
Create a CI job that stops users from adding absolute aliases (starting with slash) in localized docs

**Comment by sftim** [comment]
@rikatz
To me this is more of a backlog or long term item
/remove-priority important-soon
/priority important-longterm

and we haven't given contributors a clear steer on how to pick this up
/remove-help

**Comment by rikatz** [comment]
well, IMO saying "we need to create a CI job that checks if the aliases is not this way" was enough to have someone helping us :)

(this is why I added as help needed, not good first issue).

Anyway if no one jumps into it, I will assign myself and check during the week

**Comment by sftim** [comment]
This didn't, to me, meet our [criteria](https://www.kubernetes.dev/docs/guide/help-wanted/) for Help Wanted issues in this repo. Very happy to see the task worked on though.

If it'd be useful to transfer the issue to [k/test-infra](https://github.com/kubernetes/test-infra/), I can do that.

**Comment by divya-mohan0209** [comment]
I agree that this would be a nice-to-have CI job, however, like @sftim said I don't think it fits the help wanted label. @rikatz - are you still considering working on this?

**Comment by rikatz** [comment]
coming back to it, I am not :D sorry for the loooooong delay, forgot I have ever opened this one

**Comment by rikatz** [comment]
(feel free to close it @divya-mohan0209 !)

**Comment by divya-mohan0209** [comment]
Alright, thanks @rikatz
